docs/nix.md

```diff
@@ -0,0 +1,196 @@
+# Using Nix with Cardano SL
+
+We use Nix for building and deploying Cardano SL. It provides fully
+deterministic and cached builds of our software and all dependencies,
+and permits fully declarative configuration of the infrastructure
+running the Cardano SL network.
+
+## Installing
+
+The [Nix](https://nixos.org/nix/) package manager can be installed on
+any Linux distribution (with SELinux disabled) or on macOS.
+
+Multi user mode is the best and most commonly tested configuration. To
+install Nix in multi user mode:
+
+    sh <(curl https://nixos.org/nix/install) --daemon
+
+Full instructions are in the
+[Nix Manual](https://nixos.org/nix/manual/#ch-installing-binary).
+
+## Binary cache
+
+Configuring the IOHK binary cache is essential for developing with
+Cardano SL. If you do not have the binary cache set up correctly, Nix
+will download source tarballs and proceed to build an entire system
+from scratch.
+
+Add the following lines to `/etc/nix/nix.conf`. If the file does not
+exist, then create it.
+
+    substituters         = https://hydra.iohk.io https://cache.nixos.org
+    trusted-substituters =
```

Comment:
the emty `trusted-substituters=` isnt really needed

Reply:
The empty variable helps clear confusion that users often have about which one to use.